Updated intructions for new extension features

japere opened the pull request:
We are releasing a new update for the extension which includes new UI. The doc includes steps for using the new UI and updated browser support listings. Target for changes to go live 1/10/17-1/12/17.

azurecla commented:
Hi __@GarethBradshawMSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (garbrad). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

deneha commented:
@GarethBradshawMSFT Thanks for the contribution.  We can't have two names on the "authors" field.  You can however put your alias on the "ms.authors" field or work with @joaoma to decide if your name should be in the "authors" field instead.

japere committed:
Updated intructions for new extension features

We are releasing a new update for the extension which includes new UI. The doc includes steps for using the new UI and updated browser support listings. Target for changes to go live 1/11/17.

japere reviewed (commented):
Reviewed. Updated instructions for new extension features.
Updates should go live when feature is released. (1/10/17-1/12/17). I will update with specific date.

japere committed:
Updated publication date.

NiyaNagi reviewed (approved):
#sign-off
